.github/workflows/ci.yml

```diff
@@ -11,10 +11,6 @@ env:
   DOTNET_NOLOGO: true
 jobs:
   setup:
-    if:
```

Comment:
It’s outside the PR so I can’t comment on it but the pull_request_target trigger needs to come out.